extensions/amp-story/1.0/amp-story-quiz.css

```diff
@@ -0,0 +1,145 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins&display=swap');
+
+:host {
+    align-self: start;
+    --correct-color: #5BBA74 !important;
+    --correct-color-shaded: #2D8E42 !important;
+    --incorrect-color: #F34E4E !important;
+    --incorrect-color-shaded: #cc4141 !important;
+    --accent-color: #005AF0;
+  }
+
+.i-amp-story-quiz-container {
+    font-family: 'Poppins-bold', sans-serif !important;
+    background: var(--accent-color) !important;
+    border-radius: 32px !important;
+}
+
+.i-amp-story-quiz-prompt-container {
+    display: grid !important;
+    justify-items: center !important;
+    padding: 20px !important;
+    color: white !important;
+}
+
+.i-amp-story-quiz-prompt {
+    margin: 0px !important;
+}
+
+h1.i-amp-story-quiz-prompt {
+    font-size: 28px !important;
+    line-height: 36px !important;
+}
+
+h2.i-amp-story-quiz-prompt {
+    font-size: 22px !important;
+    line-height: 28px !important;
+}
+
+h3.i-amp-story-quiz-prompt {
+    font-size: 18px !important;
+    line-height: 24px !important;
+}
+
+.i-amp-story-quiz-option-container {
+    display: grid !important;
+    background-color: white !important;
+    border-radius: 32px !important;
+    padding: 8px !important;
+    padding-bottom: 0px !important;
+}
+
+.i-amp-story-quiz-option {
+    display: inline-grid !important;
+    grid-template-columns: 50px auto !important;
```

Comment:
Can you explain this `50px`? Is it the size of the "A/B/C/D" icon?
Does it support `RTL` out of the box (ie: will the icon be on the right of the screen?)